pkg/i2gw/notifications/config.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package notifications
+
+import (
+	"os"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
+)
+
+const (
+	bgColor = text.BgBlack
+
+	infoColor    = text.FgHiWhite
+	warningColor = text.FgHiMagenta
+	errorColor   = text.FgRed
+	defaultColor = text.FgWhite
```

Comment:
consider calling the colors by the names, not where we expect them to be used.
errorColor --> red
infoColor --> white

Reply:
i dont think i understood this comment. Do you want me to remove these and directly put the colors directly in the switch case statements?